src/js/close-button.js

```diff
@@ -0,0 +1,28 @@
+import Button from './button';
+import Component from './component';
+
+/**
+ * The `CloseButton` component is a button which fires a "close" event
+ * when it is activated.
+ *
+ * @extends Button
+ * @class CloseButton
+ */
+class CloseButton extends Button {
+
+  constructor(player, options) {
+    super(player, options);
+    this.controlText(options && options.controlText || this.localize('Close'));
+  }
+
+  buildCSSClass() {
+    return `vjs-close ${super.buildCSSClass()}`;
```

Comment:
while the close button should definitely be a button I'm a bit concerned with the fact that it gets the `vjs-control` class set on it. It currently implies a control bar control and possibly used for that places. Possibly not really an issue but wanted to call it out here.